Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py

```diff
@@ -85,12 +85,12 @@
 AutoInstall.register(Package('urllib3', Version(1, 25, 10)))
 AutoInstall.register(Package('wheel', Version(0, 35, 1)))
 AutoInstall.register(Package('whichcraft', Version(0, 6, 1)))
+AutoInstall.register(Package('cffi', Version(1, 15, 1)))
+AutoInstall.register(Package('cryptography', Version(36, 0, 2), wheel=True, implicit_deps=['cffi']))
```

Comment:
The (slightly) newer version is fine, but why have we broken this out of the version guard? The Twisted one isn't version guarded because none of that works for older versions of Python anyways, but `webkitcorepy` still does. I'm looking at breaking that assumption in the near future, but this doesn't seem like a compelling reason to break compatibility with old Python.

Reply:
Right. The bot also complained about this.

`cffi` seems to install fine on python2. so I added `cryptography` into a python3 guard

Reply:
It does! Although I need to make some Internal changes before this can land. Should be good to go in the next 24 hours, though.

Reply:
@clopez Good to land! All the Internal stuff is good to go!

Reply:
Thanks  🙏